### PR TITLE
Feat: Block alignment support for heading block

### PR DESCRIPTION
**Pull request by blankart**

### CODE REVIEW

File: src/block/heading/index.js [hunk 1]
```diff
@@ -23,6 +23,7 @@ export const settings = applyFilters( 'stackable.block.metadata', {
 	attributes: schema,
 	supports: {
 		anchor: true,
+		align: true,
```
Comment (blankart) [inline src/block/heading/index.js:26]:
```suggestion
		align: [ 'center', 'wide', 'full' ],
```

**Comment by legaspielaine** [comment]
### Issues

- [ ] When setting Heading to `Align center`, the alignment in Backend not the same in Frontend > [link](https://github.com/gambitph/Stackable/issues/2016)

**Steps to reproduce the behavior:**
1. Add Heading block
2. set this to Align center in its toolbar
3. compare the alignment in Editor vs Frontend
4. See bug - in Frontend, the Heading is left aligned but in Backend it's center aligned

**Expected behavior**
The frontend should be the same as the one in backend (center aligned).

**Screenshots**

https://user-images.githubusercontent.com/28699204/141259305-cba06010-6ff0-4b5b-84d8-166ee66255d8.mov

**Comment by legaspielaine** [comment]
this [issue](https://github.com/gambitph/Stackable/pull/2015#issuecomment-966143759) is still happening

File: src/styles/editor-block.scss [hunk 2]
```diff
@@ -310,4 +310,3 @@
 	align-items: normal;
 	justify-content: var(--stk-alignment-justify-content);
 }
```
Comment (bfintal) [inline src/styles/editor-block.scss:312]:
```suggestion
}

```

**Pull request merged**